### PR TITLE
Update docs Jun 8

### PR DESCRIPTION
**Pull request by dkluever22**
* Add Patch custom-attributes
* Udpates change log regarding 7.84 changes

### CODE REVIEW

**Review by mhoehn — APPROVED** [review]

File: _raml/apps/products/schema/patch_product_productid_customattributes.schema.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "http://json-schema.org/draft-03/schema",
+  "title": "Patch Product ProductId Custom Attributes Schema",
+
+  "type": "object",
+  "properties": {
+    "items": {
+      "type": "array of [customAttribute](page:apps-data-types#customattribute)",
+      "description": "Product-specific information that can be assigned to a product. This information can have a specific type, e.g. decimal number, can be set visible in the administration area of the shop so that it is displayed with the product and can be specified as an additional filter criterion in the shop."
```
Comment (AliceStatler) [inline _raml/apps/products/schema/patch_product_productid_customattributes.schema.json:9]: I would rephrase that a bit, especially the first sentence, to give a bit more information.

```suggestion
      "description": "Information related to a product that can be assigned to describe it in more detail. This information can have a specific type, e.g. a decimal number, and can be set to be visible in the administration area of the shop so that it is displayed with the product. It can also be specified as an additional filter criterion in the shop."
```

File: _raml/apps/products/schema/patch_product_productid_customattributes_request_schema.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "http://json-schema.org/draft-03/schema",
+  "title": "Patch Product ProductId Custom Attributes Request Schema",
```
Comment (AliceStatler) [inline _raml/apps/products/schema/patch_product_productid_customattributes_request_schema.json:3]: See comment above :)

File: _raml/apps/products/schema/patch_product_productid_customattributes_request_schema.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  "type": "object",
+  "properties": {
+    "": {
+      "type": "array of [jsonPatch](page:apps-data-types#jsonpatch)",
+      "description": ""
```
Comment (AliceStatler) [inline _raml/apps/products/schema/patch_product_productid_customattributes_request_schema.json:8]: No description?

File: apps/change-log.md [hunk 4]
```diff
@@ -16,6 +16,16 @@ Of course, we will continue to inform you if there are any breaking changes to t
...
 
+### Changes with software release 7.84.0
+
+#### <i class="fas fa-plus"></i> New
+
+* [`PATCH`/{shopId}/products/{productId}/custom-attributes](https://developer.epages.com/apps/api-reference/patch-shopid-products-productid-custom-attributes.html)
+
+<hr>
+
+## 2023-05-11
```
Comment (AliceStatler) [inline apps/change-log.md:27]: The date twice? Same as in line 17.

File: _raml/apps/products/examples/patch_product_productid_customattributes_request.json [hunk 5]
```diff
@@ -0,0 +1,20 @@
+[
+    {
+      "op": "add",
+      "path": "/",
+      "value":
+      {
+        "key": "Colour",
```
Comment (AliceStatler) [inline _raml/apps/products/examples/patch_product_productid_customattributes_request.json:7]: Don't we write the API Docs in en-US?

**Review by AliceStatler — COMMENTED** [review]
@dkluever22 Review done ✅ Please have a look at my comments and let me know if you have any questions.

File: _raml/apps/products/schema/patch_product_productid_customattributes.schema.json [hunk 6]
```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "http://json-schema.org/draft-03/schema",
+  "title": "Patch Product ProductId Custom Attributes Schema",
```
[the remark this answers is not in the document]
Comment (dkluever22) [reply _raml/apps/products/schema/patch_product_productid_customattributes.schema.json:3]: @AliceStatler Well, that's what it is :D It somehow reflects the paths of this file. Since it was done like this throughout the NOW documentation, I chose to do it this way. To be honest, I'm not sure if this is actually necessary and maybe we could change it into "Patch Custom Attributes Schema".
@mhoehn What do you think. Does anything speak against shortening the Title?

File: _raml/apps/products/schema/patch_product_productid_customattributes_request_schema.json (hunk 2 above)
Comment (dkluever22) [reply _raml/apps/products/schema/patch_product_productid_customattributes_request_schema.json:3]: @AliceStatler Let's see what @mhoehn has to say to the chase above.

File: _raml/apps/products/schema/patch_product_productid_customattributes_request_schema.json (hunk 3 above)
Comment (dkluever22) [reply _raml/apps/products/schema/patch_product_productid_customattributes_request_schema.json:8]: @AliceStatler Very often the description is left out. But I agree, let's try to do it consistently. I'll try to think of something.

File: apps/change-log.md (hunk 4 above)
Comment (dkluever22) [reply apps/change-log.md:27]: @AliceStatler Uh, nice catch. That's wrong, of course, and needs to be ## 2023-06-08

File: _raml/apps/products/examples/patch_product_productid_customattributes_request.json (hunk 5 above)
Comment (dkluever22) [reply _raml/apps/products/examples/patch_product_productid_customattributes_request.json:7]: @AliceStatler We do. Unfortunately it's not consistent throughout the NOW API docs. But you're right, we should start heading that way. I'll change that.

File: _raml/apps/products/schema/patch_product_productid_customattributes_request_schema.json (hunk 3 above)
Comment (dkluever22) [reply _raml/apps/products/schema/patch_product_productid_customattributes_request_schema.json:8]: @AliceStatler I've researched a bit in the NOW documentation and I think there is no description because there is no actual property that might be described. It's just a schema that shows how the structure would look like without values. So I think this has to be this way.
@mhoehn Please correct me if I'm wrong ;)

File: _raml/apps/products/schema/patch_product_productid_customattributes.schema.json (hunk 6 above)
[the remark this answers is not in the document]
Comment (mhoehn) [reply _raml/apps/products/schema/patch_product_productid_customattributes.schema.json:3]: @AliceStatler "Patch Custom Attributes Schema" is ok

File: _raml/apps/products/schema/patch_product_productid_customattributes_request_schema.json (hunk 3 above)
Comment (mhoehn) [reply _raml/apps/products/schema/patch_product_productid_customattributes_request_schema.json:8]: @dkluever22 Correct: "It's just a schema that shows how the structure would look like without values."

**Comment by dkluever22** [comment]
@AliceStatler @mhoehn If you both could have a final look at the PR and approve, if not already done. Thanks in advance!

File: _raml/apps/products/schema/patch_product_productid_customattributes.schema.json (hunk 6 above)
[the remark this answers is not in the document]
Comment (AliceStatler) [reply _raml/apps/products/schema/patch_product_productid_customattributes.schema.json]: @dkluever22 Then go for it!

File: _raml/apps/products/schema/patch_product_productid_customattributes_request_schema.json (hunk 3 above)
Comment (AliceStatler) [reply _raml/apps/products/schema/patch_product_productid_customattributes_request_schema.json:8]: @dkluever22 Okay :) I think it's fine without the description. I was just wondering.

**Review by AliceStatler — APPROVED** [review]
@dkluever22 There you go
